FluentAvalonia/Styling/Core/FluentAvaloniaTheme.axaml

```diff
@@ -0,0 +1,16 @@
+﻿<Styles
+    xmlns="https://github.com/avaloniaui"
```

Comment:
nit: xmlns on the same lines as styles, then align the others with that

Reply:
Done.